apollo-runtime/src/main/java/com/apollographql/android/impl/RealApolloCall.java

```diff
@@ -133,7 +137,10 @@ private RealApolloCall(Operation operation, HttpUrl serverUrl, Call.Factory http
       throw new HttpException(response);
     } else {
       try {
-        return responseBodyConverter.convert(response.body());
+        ResponseNormalizer normalizer = cache.responseNormalizer();
+        Response<T> convertedResponse = responseBodyConverter.convert(response.body(), normalizer);
+        cache.cacheStore().merge(normalizer.records());
```

Comment:
So the question is if someone has a faulty `cacheStore`, is it better to fail the entire response, or catch the error?

My issue is if we catch the error, how to we communicate that the `cacheStore` is faulty?